core/ustring.cpp

```diff
@@ -4214,6 +4214,45 @@ String String::unquote() const {
 	return substr(1, length() - 2);
 }
 
+String String::check_control_characters() const {
+	String s = *this;
+	String slash = "\\";
+	String tab = "\t";
+	size_t tab_pos = s.find(tab, 0);
+
+	while (tab_pos != npos) {
+		String str_begin = s.substr(0, tab_pos);
+		String str_end = s.substr(tab_pos + 1, s.length());
+
+		s = str_begin + "    " + str_end;
```

Comment:
Should we always assume, that \t == 4 spaces? Shouldn't it be configurable per usecase?